src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -190,16 +216,15 @@ bool emberAfGroupsClusterAddGroupCallback(app::CommandHandler * commandObj, cons
 bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, const app::ConcreteCommandPath & commandPath,
                                            const Commands::ViewGroup::DecodableType & commandData)
 {
-    auto & groupId = commandData.groupId;
-
-    EmberAfStatus status                                          = EMBER_ZCL_STATUS_NOT_FOUND;
-    CHIP_ERROR err                                                = CHIP_NO_ERROR;
-    uint8_t groupName[ZCL_GROUPS_CLUSTER_MAXIMUM_NAME_LENGTH + 1] = { 0 };
+    auto fabricIndex = GetFabricIndex(commandObj);
+    auto & groupId   = commandData.groupId;
```

Comment:
Is there a reason why this is a reference rather than a value? Same for all instances of `commandData.groupId`.